README.md

```diff
@@ -4,11 +4,11 @@ Builds an image of the Apache qpid dispatch router designed for use with kuberne
 e.g. to build:
 
 ```
-make && docker build -t quay.io/interconnectedcloud/qdrouterd:1.8.0 . && docker push quay.io/interconnectedcloud/qdrouterd:1.8.0
+make && docker build -t quay.io/interconnectedcloud/qdrouterd:latest . && docker push quay.io/interconnectedcloud/qdrouterd:latest
 ```
```

Comment:
I think we want to tag with the actual version *as well* as with latest. So users can take latest or use specific version if desired.

Reply:
Actually, thinking a bit more, this is just a readme, so comments above don't really apply here.